qemu.go

```diff
@@ -243,6 +243,7 @@ type Config struct {
 	CharDevices           []CharDevice           `yaml:"char-devices"`
 	LegacySerialDevices   []LegacySerialDevice   `yaml:"legacy-serial-devices"`
 	SerialDevices         []SerialDevice         `yaml:"serial-devices"`
+	PCISerialDevices 	  []PCISerialDevice	     `yaml:"pci-serial-devices"`
```

Comment:
We don't need this, since the SerialDevice already contains a DeviceDriver object, so we can create a SerialDevice which uses Driver "pci-serial".

Reply:
Take a look at serialdevice_test.go for the VirtioSerial case;  I think the PCISerial should look very similar to that (ignoring the extra options that a VirtioSerial device has)

```
$ qemu-system-x86_64 --device pci-serial,?
pci-serial options:
  x-pcie-lnksta-dllla=<bool> - on/off
  failover_pair_id=<str>
  chardev=<str>          - ID of a chardev to use as a backend
  romfile=<str>
  rombar=<uint32>
  addr=<int32>           - Slot and optional function number, example: 06.0 or 06
  prog_if=<uint8>
  x-pcie-extcap-init=<bool> - on/off
  command_serr_enable=<bool> - on/off
  multifunction=<bool>   - on/off
```

Reply:
Ah yep sorry, didn't see that the `ID` attribute of `SerialDevice` is the associated serial device